ext/django_stubs_ext/patch.py

```diff
@@ -81,16 +91,36 @@ def __repr__(self) -> str:
     # These types do have native `__class_getitem__` method since django 4.1:
     MPGeneric(ForeignKey, (4, 1)),
     MPGeneric(RawQuerySet),
+    MPGeneric(classproperty),
+    MPGeneric(ConnectionProxy),
+    MPGeneric(ModelFormOptions),
+    MPGeneric(Options),
+    MPGeneric(BaseIterable),
+    MPGeneric(ForwardManyToOneDescriptor),
+    MPGeneric(ReverseOneToOneDescriptor),
 ]
 
 
+def _get_need_generic() -> list[MPGeneric[Any]]:
+    try:
+        from django.contrib.auth.forms import SetPasswordMixin, SetUnusablePasswordMixin
+
+        return [MPGeneric(SetPasswordMixin), MPGeneric(SetUnusablePasswordMixin), *_need_generic]
+
+    except (ImproperlyConfigured, AppRegistryNotReady):
+        # logger.warning(
+        #     "`SetPasswordMixin` and `SetUnusablePasswordMixin` where not patched, django need to be configured "
+        #     "for this to be possible. Make sure to use `django_stubs_ext.monkeypatch()` after `django.setup()`.",
+        # )
```

Comment:
This is annoying, we cannot easily patch these 2 because we end up [importing the User model](https://github.com/django/django/blob/main/django/contrib/auth/forms.py#L7) and it crashes if `django.setup()` was not called beforehand.

The recommendation from the readme to add the monkeypatch in the setting file might not be enough for this.

I commented the logs here for the moment because I'm not sure what is the best way to handle this (and if I add it, I need to find a way to silence this logger in test or it fails some snapshot)

Any idea on how to handle this @sobolevn ? (or @adamchainz maybe if you have some time)

Reply:
I've documented a workaround for these symbols that seem to work on my django codebases. 

If you use these generics, you'll have to rerun the monkeypatch after django is initialized.
We could also maybe expose another `monkeypatch` func for these